packages/documentation/copy/en/handbook-v2/Everyday Types.md

```diff
@@ -141,7 +141,7 @@ Here's an example:
 
 ```ts twoslash
 // @errors: 2551
-// No type annotations here, but TypeScript can spot the bug
+// No type annotations here, but TypeScript can infer the constant to be string[]

```

Comment:
This section to me reads to be talking about function parameter type inference, e.g. that `s` in the two functions below get type `string`. But that has nothing to do with this line having an annotation or not. If this line had an annotation, the contextual typing below still occurs.

So honestly, I think maybe the right thing here is to drop this line altogether.

Reply:
I removed the comment about that line, and made it more explicit what the example was showing by adding to the next comment. Hope it's better.